### PR TITLE
Update opentelemetry-proto dependency and the internal metric definition

### PR DESCRIPTION
**Pull request by bogdandrutu**
The copyright was added automatically when run `make`

### CODE REVIEW

File: translator/trace/oc_const.go [hunk 1]
```diff
@@ -1,3 +1,17 @@
+// Copyright 2019 OpenTelemetry Authors
```
Comment (pjanotti) [inline translator/trace/oc_const.go]: nit: 2020 but I guess we need to update the tool

Comment (bogdandrutu) [reply translator/trace/oc_const.go]: Done manually, please update the tool if you know where it is.

File: go.sum [hunk 2]
```diff
@@ -511,6 +511,8 @@ github.com/open-telemetry/opentelemetry-proto v0.0.0-20200211051721-ff5f19c6217d
 github.com/open-telemetry/opentelemetry-proto v0.0.0-20200211051721-ff5f19c6217d/go.mod h1:PMR5GI0F7BSpio+rBGFxNm6SLzg3FypDTcFuQZnO+F8=
 github.com/open-telemetry/opentelemetry-proto v0.0.0-20200227223449-a65b867510c0 h1:g8x0BMEHMKlLYJEYj0SucilB796mVMcef/xjnd/rH8E=
 github.com/open-telemetry/opentelemetry-proto v0.0.0-20200227223449-a65b867510c0/go.mod h1:PMR5GI0F7BSpio+rBGFxNm6SLzg3FypDTcFuQZnO+F8=
+github.com/open-telemetry/opentelemetry-proto v0.0.0-20200308012146-674ae1c8703f h1:o21WlujGsrjoN3+n99AflASF/K0S6+SOLOj2O9nNplc=
+github.com/open-telemetry/opentelemetry-proto v0.0.0-20200308012146-674ae1c8703f/go.mod h1:PMR5GI0F7BSpio+rBGFxNm6SLzg3FypDTcFuQZnO+F8=
```
Comment (tigrannajaryan) [inline go.sum]: Please run `go mod tidy` to prune old references if you didn't already.

Comment (bogdandrutu) [reply go.sum]: done.

File: translator/trace/oc_const.go (hunk 1 above)
Comment (pjanotti) [reply translator/trace/oc_const.go]: https://github.com/open-telemetry/opentelemetry-collector/pull/604

**Pull request merged**